lib/govuk_content_models/presentation_toggles.rb

```diff
@@ -0,0 +1,39 @@
+module PresentationToggles
+  extend ActiveSupport::Concern
+
+  included do
+    field :presentation_toggles, type: Hash, default: default_presentation_toggles
+    validates_presence_of :organ_donor_registration_promotion_url, if: :display_organ_donor_registration_promotion?
+  end
+
+  def display_organ_donor_registration_promotion=(value)
+    value = value.is_a?(Boolean) ? value : value != '0' # if assigned using a checkbox
+    organ_donor_registration_promotion_key['display'] = value
+  end
+
+  def display_organ_donor_registration_promotion
+    organ_donor_registration_promotion_key['display']
+  end
+  alias_method :display_organ_donor_registration_promotion?, :display_organ_donor_registration_promotion
+
+  def organ_donor_registration_promotion_url=(value)
+    organ_donor_registration_promotion_key['url'] = value
+  end
+
+  def organ_donor_registration_promotion_url
+    return "" unless display_organ_donor_registration_promotion?
+    organ_donor_registration_promotion_key['url']
+  end
+
+  def organ_donor_registration_promotion_key
+    presentation_toggles['organ_donor_registration_promotion']
+  end
+
+  module ClassMethods
+    def default_presentation_toggles
+      {
+        'organ_donor_registration_promotion' => { 'url' => '', 'display' => false }
+      }
+    end
+  end
+end
```

Comment:
@danielroseman: the embedded document needs fields to be defined upfront, and as i see it, with presentation toggles it could just be a boolean, like for beta label, or boolean and url like the one we have here. using a `Hash` field gives us that flexibility, reduces all this information to a single field, and helps keep content-api immune to these changes ([it just has to forward all presentation toggles](https://github.com/alphagov/govuk_content_api/pull/211/files#diff-cc4a1844299ac5a77a45849ce8471bf9R41)).